test/types/range/unbounded/unboundedBoolEnum.chpl

```diff
@@ -0,0 +1,33 @@
+enum color { red, green, blue };
```

Comment:
From a whitebox testing perspective, I'm confident that that won't cause problems, but since the code could change in the future, I agree that adding a test to defensively lock the current behavior would be good.  I've got one ready to go on my next commit.

Reply:
Added test in https://github.com/chapel-lang/chapel/pull/19991/commits/0171a3ec67aa2a9e2c9c332fd2f1287877ff91cf